src/main/java/com/uber/cadence/internal/worker/ActivityWorker.java

```diff
@@ -155,43 +150,52 @@ private TaskHandlerImpl(ActivityTaskHandler handler) {
     }
 
     @Override
-    public void handle(MeasurableActivityTask task) throws Exception {
+    public void handle(PollForActivityTaskResponse task) throws Exception {
       Scope metricsScope =
           options
               .getMetricsScope()
               .tagged(
-                  ImmutableMap.of(MetricsTag.ACTIVITY_TYPE, task.task.getActivityType().getName()));
+                  ImmutableMap.of(
+                      MetricsTag.ACTIVITY_TYPE,
+                      task.getActivityType().getName(),
+                      MetricsTag.WORKER_TYPE,
```

Comment:
Should this be `WorkflowType` or `WORKER_TYPE`?

Reply:
Yes it should be workflow type. Fixed.